drivers/hmis/app/models/hmis/form/form_processor.rb

```diff
@@ -222,15 +227,13 @@ def client_factory(create: true) # rubocop:disable Lint/UnusedMethodArgument
     end
   end
 
-  # Common HUD Assessment-related attributes
+  # Common HUD Assessment-related attributes. These always have enrollments so enrollment_factory *should* exist
   def common_attributes
     data_collection_stage = owner.data_collection_stage if owner.respond_to?(:data_collection_stage)
-    personal_id = owner.personal_id if owner.respond_to?(:personal_id)
-    information_date = owner.information_date if owner.respond_to?(:information_date)
+    personal_id = enrollment_factory(create: false)&.personal_id
     {
       data_collection_stage: data_collection_stage,
```

Comment:
@martha to your other note, Data Collection Stage is required for Disability records, so we need to set it somehow. maybe form submissions can just have a method for dcs that returns 1 (intake).